.travis.yml

```diff
@@ -5,9 +5,23 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
```

Comment:
Personally I would prefer testing all versions we support to ensure we don't run into weird issues. We also might want to use the new cron feature to schedule builds, but that only makes sense with `nightly` tbh.